src/Billing/StatementsClient.php

```diff
@@ -114,7 +114,7 @@ class StatementsClient extends AbstractBillingClient
     public const FORMAT = 'format';
 
     /**
-     * @var string per page default value
+     * @var integer per page default value
```

Comment:
```suggestion
     * @var int per page default value
```
integer ain't a valid PHP type